pkg/operator/k8sutil/network.go

```diff
@@ -26,6 +26,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// PublicNetworkSelectorKeyName is the network selector key for the ceph public network
+	PublicNetworkSelectorKeyName = "public"
```

Comment:
This should not move here since the k8sutil package is generic for all the operators, not specific to Ceph.